src/components/Dropzone/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { ChangeEvent, useRef } from 'react'
+import styled from '@emotion/styled'
+import { color, space, radius, fontWeight } from '../../theme'
+import { H4 } from '../Heading'
+import { SmallText } from '../Text'
+import { BaseButton } from '../BaseButton'
+import ILLUSTRATION from './files.svg'
+
+interface DropzoneProps {
+  title: string
+  subtitle: string
+  action: string
+  ariaLabel?: string
+  accept?: Array<string>
+  onFileChange: Function
+  onUnacceptedFileChange?: Function
+}
+
+const DropArea = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  background-color: ${color.actionBackground};
+  border-radius: ${radius.lg};
+  padding-block-start: ${space[16]};
+  padding-block-end: ${space[24]};
+  padding-inline: ${space[16]};
+  text-align: center;
+  position: relative;
+  border: 2px dashed ${color.earth};
+`
+
+const Illustration = styled.img`
+  width: 110px;
+  height: 75px;
+  cursor: pointer;
+`
+
+const Title = styled(H4)`
+  font-weight: ${fontWeight.semiBold};
+`
+
+const Information = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  padding-block-start: ${space[12]};
+  flex: 1;
+  gap: ${space[8]};
+`
+
+const FileInput = styled.input`
+  display: none;
+`
+
+export const Dropzone = ({
+  title,
+  subtitle,
+  action,
+  accept = ['*'],
+  ariaLabel,
+  onFileChange,
+}: DropzoneProps) => {
+  const inputFile = useRef<HTMLInputElement | null>(null)
+
+  const onSelectFile = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e?.target?.files?.length) {
+      onFileChange(e.target.files[0])
+    }
+  }
+
+  return (
+    <DropArea data-testid="droparea">
+      <>
+        <Illustration src={ILLUSTRATION} />
```

Comment:
Images need width and height property on it, to calculate size and aspect ration before css is loaded (always the actual size of the image). Also an alt is needed, `alt=""` for images without meaning.

Reply:
Interesting! What happens if we don't do the calculation with CSS? Will it display the image in fullsize until the Javascript is loaded in the client side? 👀

Reply:
Fixed btw 🙏